src/utils/util.test.ts

```diff
@@ -0,0 +1,295 @@
+import axios from 'axios';
+import { describe, it, expect, vi } from 'vitest';
+import { mockGitHubResponse, mockConfigFile, mockTermsData, mockConfig } from './mocks';
+import { fetchAvailableConfigs, fetchConfig, mapConfigFileToStoreConfig } from './util';
+
+// Mock axios
+vi.mock('axios');
+const mockedAxios = vi.mocked(axios, true);
+
+describe('fetchAvailableConfigs', () => {
+  it('should fetch available configs from GitHub successfully', async () => {
+    mockedAxios.get.mockResolvedValueOnce({ data: mockGitHubResponse });
+
+    const result = await fetchAvailableConfigs();
+
+    expect(mockedAxios.get).toHaveBeenCalledWith(
+      'https://api.github.com/repos/neurobagel/communities/contents/'
```

Comment:
Would be nice to factor out these URLs into a couple of constants at the top of the file, so we could more easily adjust them later if things change